### DISCUSSION TITLE
Option for inserting pairs only if there's whitespace

### DISCUSSION THREAD

**Discussion by tonekk** [LunarWatcher/auto-pairs#63] [Q&A] [answered]
Where can I find that option?

Do I get that right, like inserting a `"` before a char would not create a pair?
Would love that :)

Thanks

**Comment by LunarWatcher** [comment, marked as answer]
It's listed in the docs; you're looking for `g:AutoPairsCompleteOnlyOnSpace`. See `:h g:AutoPairsCompleteOnlyOnSpace` for its documentation, and `h autopairs-options-contents` for a brief overview of all the config variables available.
